stanford-sw/test/src/edu/stanford/ManagedPurlTests.java

```diff
@@ -32,46 +32,60 @@ public final void setup()
 	 * test managed_purl_urls field
 	 */
 @Test
-	public final void testManagedPurlUrls() 
+	public final void testManagedPurlUrls()
 			throws ParserConfigurationException, IOException, SAXException
 	{
 		String fldName = "managed_purl_urls";
-			
-		assertDocHasFieldValue("managedPurlItem1Collection", fldName, "https://purl.stanford.edu/nz353cp1092"); 
-		assertDocHasFieldValue("managedPurlItem3Collections", fldName, "https://purl.stanford.edu/wd297xz1362"); 
-		assertDocHasFieldValue("managedPurlCollection", fldName, "https://purl.stanford.edu/ct961sj2730"); 
-		assertDocHasFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "https://purl.stanford.edu/ct961sj2730"); 
-		
+
+		assertDocHasFieldValue("managedPurlItem1Collection", fldName, "https://purl.stanford.edu/nz353cp1092");
+		assertDocHasFieldValue("managedPurlItem3Collections", fldName, "https://purl.stanford.edu/wd297xz1362");
+		assertDocHasFieldValue("managedPurlCollection", fldName, "https://purl.stanford.edu/ct961sj2730");
+		assertDocHasFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "https://purl.stanford.edu/ct961sj2730");
+
 		assertDocHasNoField("NoManagedPurlItem", fldName);
 		assertDocHasNoFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "https://purl.stanford.edu/yy000zz0000");
 	}
-	
+
 	/**
 	 * Test method for display_type.
 	 */
 @Test
-	public final void testDisplayType() throws IOException, ParserConfigurationException, SAXException 
+	public final void testDisplayType() throws IOException, ParserConfigurationException, SAXException
 	{
 		String fldName = "display_type";
-	
-		assertDocHasFieldValue("managedPurlItem3Collections", fldName, "file"); 
-		assertDocHasFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "image"); 
-		
+
+		assertDocHasFieldValue("managedPurlItem3Collections", fldName, "file");
+		assertDocHasFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "image");
+
 		assertDocHasFieldValue("NoManagedPurlItem", fldName, "sirsi");
 		assertDocHasNoFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "book");
 	}
-	
+
+/**
+ * Test method for fileId.
+ */
+@Test
+public final void testSetFileId() throws IOException, ParserConfigurationException, SAXException
+{
```

Comment:
most of the time I don't care much about alignment, but left justified for this feels uncomfy.